docs/en/4-0-upgrade-guide.rst

```diff
@@ -14,3 +14,24 @@ Breaking Changes
 * ``Query::isEagerLoaded()``, and ``Query::eagerLoaded()`` were removed.
   Previously these methods were inherited from ``QueryTrait`` but served no
   purpose here.
+
+
+Indexes
+================
```

Comment:
```suggestion
======
```

Reply:
> RST doesn't compile if the headings are not the same lenght as the text above

Small correction, title underlines can be equal length or longer. Short underlines are the problematic scenario.